src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/MetadataLoadContext.Loading.cs

```diff
@@ -34,7 +34,7 @@ private RoAssembly LoadFromStreamCore(Stream peStream)
                 byte[] pkt = defNameData.PublicKeyToken ?? Array.Empty<byte>();
                 if (pkt.Length == 0 && defNameData.PublicKey != null && defNameData.PublicKey.Length != 0)
                 {
-                    pkt = defNameData.PublicKey.ComputePublicKeyToken();
+                    pkt = defNameData.PublicKey!.ComputePublicKeyToken()!;
```

Comment:
Is the 1st bang, after defNameData.PublicKey needed here?

Reply:
Yes, PublicKey is nullable.

Reply:
Oh, I hadn't realized that `!` forces every node in an expression.